code/modules/spells/spell_types/wizard.dm

```diff
@@ -261,17 +261,65 @@
 	invocation_type = "shout"
 	range = 20
 	cooldown_min = 20 //10 deciseconds reduction per rank
-
+	var/fireball_type = /obj/item/projectile/magic/fireball

```

Comment:
this is probably a nerf because this won't check around it for targets to explode on and is thus, with the changed aiming, way harder to hit with